sportsdataverse/nba/nba_model_validation.py

```
@@ -0,0 +1,808 @@
+            uncertainty set it (enables interval calibration). ``None`` for point models.
+    """
+
+    coef: np.ndarray
+    intercept: float
+    posterior: Optional[np.ndarray] = None
+
+
+class RapmModel(Protocol):
+    """Design-matrix estimator: fit a sparse design ``X`` against targets ``y``."""
+
+    def fit(self, X: csr_matrix, y: np.ndarray) -> FitResult: ...
```

Flake8 (7.3.0)

[error] multiple statements on one line (def)

(E704)